microsoft/PowerToys
# Remove VCRedist Merge Modules, copy dlls instead

* VC++ redistributables for CRT are copied into \PowerToys, \PowerToys\SettingsUIRunner and \PowerToys\modules\launcher
* This is intended to solve an issue where Firefox would fail because of missing dlls after uninstalling PowerToys. I couldn't reproduce the issue personally, Firefox kept working properly when I tried.
* This affects PowerToys binaries compiled with the \MD flag, which are PowerToysInterop and os-detection
* This is a hotfix, and we should probably include VCRedist_64.exe in our installer as recommended by MSDN https://docs.microsoft.com/en-us/cpp/windows/determining-which-dlls-to-redistribute?view=vs-2019, which would also avoid having the DLLs copied multiple times in various folders.
* Since this adds new dlls, it might affect the resolution of the DLLs signing issue.
* VC++ binaries copied from my copy of Visual Studio 2019, they are on \Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Redist\MSVC\14.25.28508\x64\Microsoft.VC142.CRT 



## PR Checklist


## Validation Steps Performed
1. Install Power Toys in as Fresh a Windows installation as possible (to ensure some other application doesn't install the Visual C++ Redistributables)
2. Start Power Toys
3. Open Settings. Shouldn't crash.
4. Open Launcher. Shouldn't crash.
